framework/doc/content/source/userobjects/NodalPatchRecoveryVariable.md

```diff
@@ -0,0 +1,13 @@
+# NodalPatchRecoveryVariable
+
+This user object performs the calculations and parallel communication required to carry out the [Zienkiewicz–Zhu patch recovery](https://doi.org/10.1002/nme.1620330702) for a scalar component of a provided variable.
```

Comment:
Thanks for adding the reference! However, please instead use the [!cite] syntax to reference the paper, and then add an entry for it in one of the .bib files.